assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -294,9 +311,49 @@ const SearchFunnelWidget = ( {
 	}
 
 	if ( isSearchConsoleGatheringData ) {
+		const halfCellProps = {
+			smSize: 4,
+			mdSize: 4,
+			lgSize: 6,
+		};
+
 		return (
 			<Widget Header={ Header } Footer={ WidgetFooter }>
-				<WidgetReportZero moduleSlug="search-console" />
+				{ isAnalyticsConnected && isAnalyticsActive && (
```

Comment:
Connected will only be true if the module is also active so we don't need to check both here. https://github.com/google/site-kit-wp/blob/8b08f895c24403d4f7ae844f87a03dd88691098a/assets/js/googlesitekit/modules/datastore/modules.js#L787